wordpress-mobile/WordPress-iOS
# Stats Insights tweaks – default cards, modal presentation

Fixes #13127. This PR makes a few updates to Stats Insights:

* The default set of cards has now been updated to match the issue linked above.
* The **Add stats card** view is now presented modally instead of pushed

**Default cards**

<img width="414" alt="Simulator Screen Shot - iPhone 11 - 2019-12-17 at 17 28 58" src="https://user-images.githubusercontent.com/4780/71027527-57cea380-2103-11ea-8691-d7959d6e0051.png">

**Modal presentation**

<img width="414" alt="Simulator Screen Shot - iPhone 11 - 2019-12-17 at 17 29 02" src="https://user-images.githubusercontent.com/4780/71027541-5c935780-2103-11ea-9970-cb245640cf7a.png">

**To test:**

* Build and run
* With a site you haven't accessed before or modified insights, check that the default insights list is:
   * Latest Post Summary
   * Today
   * All-Time
   * Follower Totals
* Tap **Add stats card** and check that the view is presented modally and still functions as expected.
* Also test on iPad

PR submission checklist:
